packages/cli-kit/src/private/node/ui/hooks/use-layout.ts

```diff
@@ -0,0 +1,23 @@
+import {useStdout} from 'ink'
+
+const MIN_WIDTH = 80
+
+export default function useLayout() {
```

Comment:
What about adding a unit test for this one?

Reply:
Will do 👍